analyzers/src/SonarAnalyzer.CFG/Roslyn/ControlFlowGraph.cs

```diff
@@ -60,7 +60,7 @@ public class ControlFlowGraph
 
         static ControlFlowGraph()
         {
-            if (RoslynHelper.FlowAnalysisType("ControlFlowGraph") is { } type)
+            if (RoslynHelper.RoslynVersionIsSupported() && RoslynHelper.FlowAnalysisType("ControlFlowGraph") is { } type)
             {
                 IsAvailable = true;
```

Comment:
We actually don't need the 2nd part as a condition. The new version check is all that matters.
```suggestion
            if (RoslynHelper.RoslynVersionIsSupported() && RoslynHelper.FlowAnalysisType("ControlFlowGraph") is { } type)
            {
                IsAvailable = true;
                var type = RoslynHelper.FlowAnalysisType("ControlFlowGraph");
```